### PR TITLE
feat(CORSO-36): modify a review

### PR DESCRIPTION
**Pull request by DylanBrass**
JIRA: [link to jira ticket](https://champlainsaintlambert.atlassian.net/browse/CORSO-36?atlOrigin=eyJpIjoiZWZkNGI3NmY3NTRhNDU4MmJjNWViYmRkZGJiMGQ2MjMiLCJwIjoiaiJ9)
## Context:
Given that I am a customer who has previously posted a review, When I navigate to my review and click edit, Then I'm able to make changes to my review. This should send an email to the admin if the review was pinned on the main page.
## Changes
- Added localisation
- Added ability to modify an unpinned review
### Before

image

### After

image

image

### CODE REVIEW

**Review by KarinaSofia — DISMISSED** [review]
good work on your feature and good work on localization!

**Review by KarinaSofia — APPROVED** [review]
good work on your feature and good work on localization!

File: corso_website_fe/src/Pages/MainPage/Main Page Components/Services/AllServices.tsx [hunk 1]
```diff
@@ -30,7 +33,7 @@ function AllServices() {
     return (
         <div className="container services-container">
             <div className="row service-header">
-                <h1 className="service-title">All Services</h1>
+                <h1 className="service-title">{t("services")}</h1>
```
Comment (Mila5847) [inline corso_website_fe/src/Pages/MainPage/Main Page Components/Services/AllServices.tsx:36]: So you are using the useTranslation here to put the services either in French or in English right?

**Review by Mila5847 — APPROVED** [review]
The modify review looks good, and localization looks nice

File: corso_website_fe/src/Pages/MainPage/Main Page Components/Services/AllServices.tsx (hunk 1 above)
Comment (DylanBrass) [reply corso_website_fe/src/Pages/MainPage/Main Page Components/Services/AllServices.tsx:36]: Yes

**Pull request merged**